control/table_test.go

```diff
@@ -81,4 +81,31 @@ var _ = Describe("Table", func() {
 			Expect(table.Content()).To(ContainSubstring(`Responsible Role: Amazon Elastic Compute Cloud: AWS Staff`))
 		})
 	})
+	Describe("Diff", func() {
+		It("detects no diff when the value of responsible role is empty", func() {
+			doc := docFixture("AC-2")
+			tables, _ := doc.Search("//w:tbl")
+			table := tables[0]
+
+			ct := Table{Root: table}
+			openControlData := openControlFixture()
+			diff, err := ct.Diff(openControlData)
+
+			Expect(diff).To(Equal([]string{}))
+			Expect(err).To(BeNil())
+		})
+		It("detects a diff when the value of responsible role is different from `Amazon Elastic Compute Cloud: AWS Staff`", func() {
+			Skip("will fix soon")
+			doc := docFixture("AC-2")
+			tables, _ := doc.Search("//w:tbl")
+			table := tables[0]
+
+			ct := Table{Root: table}
+			openControlData := openControlFixture()
+			diff, err := ct.Diff(openControlData)
+
+			Expect(diff).ToNot(Equal([]string{}))
```

Comment:
Maybe we return something a bit more structured - e.g. keyed by the field that differs - so that we can have more specific tests?